File: hw/vargas/u2/hw19Inheritance/python/model/Pig.py
from model.FarmAnimal import FarmAnimal
class Pig(FarmAnimal):
    def __init__(self, id, breed, born_on, gender, is_able_to_reproduce, weight, cage):
        super().__init__(id, breed, born_on, gender, is_able_to_reproduce, weight, cage)

    def __str__(self):
        return f"Pig{{, {super().__str__()}}}"

File: hw/vargas/u2/hw19Inheritance/js/model/Pig.js
const FarmAnimal = require('./FarmAnimal');

class Pig extends FarmAnimal {
    constructor(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location) {
        super(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location);
    }

    toString() {
        return `Pig{, ${super.toString()}}`;
    }
}

module.exports = Pig;